app/models/manageiq/providers/kubevirt/inventory/parser.rb

```diff
@@ -79,7 +79,7 @@ def process_node(object)
     # Add the inventory object for the host:
     host_object = host_collection.find_or_build(uid)
     host_object.connection_state = 'connected'
-    host_object.ems_cluster = cluster_collection.lazy_find(CLUSTER_ID)
+    host_object.cluster = cluster_collection.lazy_find(CLUSTER_ID)
```

Comment:
Hm I don't think this will work, the association is still [`belongs_to :ems_cluster`](https://github.com/ManageIQ/manageiq/blob/master/app/models/host.rb#L32)

Reply:
Yeah,
```
  1) ManageIQ::Providers::Kubevirt::Inventory::Parser::FullRefresh#parse works correctly with one node
     Failure/Error: host_object.cluster = cluster_collection.lazy_find(CLUSTER_ID)
     
     NoMethodError:
       undefined method `cluster=' for #<#<Class:0x00005594b207cf78>:0x00005594b22c6a18>
     # ./app/models/manageiq/providers/kubevirt/inventory/parser.rb:82:in `process_node'
```